### ISSUE TITLE
[1.0] Errors on introspectionQuery

### ISSUE THREAD

**Issue by fabien0102** [gatsbyjs/gatsby#1194]
## Context

To have a great DX, I have an awesome npm command (https://github.com/fabien0102/gatsby-starter/blob/master/package.json#L20 ) to generate all graphql types for typescript (https://github.com/fabien0102/gatsby-starter/blob/master/src/graphql-types.d.ts).

## Problem

Since my update from gatsby@1.0.0-alpha13 to gatsby@1.0.0-beta.2, this tool (and any tools that use the graphql introspectionQuery) doesn't work anymore…

## Details of my debug

- The famous "instrospectionQuery"
https://github.com/graphql/graphql-js/blob/v0.4.12/src/utilities/introspectionQuery.js#L11-L89

- The generated error

```graphql
"errors": [
    {
      "message": "Int cannot represent non 32-bit signed integer value: (empty string)",
      "locations": [
        {
          "line": 57,
          "column": 5
        }
      ],
      "path": [
        "__schema",
        "types",
        158,
        "fields",
        4,
        "args",
        7,
        "defaultValue"
      ]
    },
    {
      "message": "Int cannot represent non 32-bit signed integer value: (empty string)",
      "locations": [
        {
          "line": 57,
          "column": 5
        }
      ],
      "path": [
        "__schema",
        "types",
        158,
        "fields",
        5,
        "args",
        7,
        "defaultValue"
      ]
    },
    {
      "message": "Int cannot represent non 32-bit signed integer value: (empty string)",
      "locations": [
        {
          "line": 57,
          "column": 5
        }
      ],
      "path": [
        "__schema",
        "types",
        158,
        "fields",
        6,
        "args",
        10,
        "defaultValue"
      ]
    }
  ]
```

- The error field (The query works if I comment this line)
https://github.com/graphql/graphql-js/blob/v0.4.12/src/utilities/introspectionQuery.js#L70

**Comment by KyleAMathews** [comment]
What's the field that's causing the trouble?

**Comment by fabien0102** [comment]
It's `defaultValue` (https://github.com/graphql/graphql-js/blob/v0.4.12/src/utilities/introspectionQuery.js#L70)

**Comment by KyleAMathews** [comment]
@fabien0102 want to try again with beta.6? I just tried the introspection query in a beta.6 site and it worked just fine. It could also be a bug that's only triggered on some sites not others.

**Comment by fabien0102** [comment]
@KyleAMathews Same error for me on beta.6, and the website is broken (but graphiql is working)

**Comment by KyleAMathews** [comment]
Fixed in #1352
